RELEASING.md

```diff
@@ -1,3 +1,74 @@
 ## Releasing
 
-This project follows [OpenSearch project branching, labelling, and releasing](https://github.com/opensearch-project/.github/blob/main/RELEASING.md).
\ No newline at end of file
+This project follows [OpenSearch project branching, labelling, and releasing](https://github.com/opensearch-project/.github/blob/main/RELEASING.md).
+
+## Runbook
+
+### Overview
+
+The OpenSearch project releases versioned distributions of OpenSearch, OpenSearch Dashboards, and the OpenSearch plugins. This runbook describes in detail how to perform an end-to-end major/minor/patch version release for the OpenSearch Dashboards project as the release manager (RM) of this release version. RM is responsible for updating the release status broadly on the release tracking issue ([example](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2230)) on GitHub.
+
+\*Important Dates: https://opensearch.org/releases.html
+
+### Release Phase 1 - Preparation
+
+For major and minor releases, the OpenSearch build repository [maintainers](https://github.com/opensearch-project/opensearch-build/blob/main/MAINTAINERS.md) will create a release issue in the OpenSearch Dashboards repository ([example](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2230)) which links to the overall issue in the OpenSearch build repository ([example](https://github.com/opensearch-project/opensearch-build/issues/2447)). For patch releases, they will only create the overall issue in the OpenSearch build repository ([example](https://github.com/opensearch-project/opensearch-build/issues/2650)).
+
+The release manager (RM) will be assigned to the OpenSearch Dashboards release issue and will be responsible for reviewing all tasks in this issue for preparation. It's also a good idea to compare the current release issue to the issue of previous release version at this point to ensure that all new processes have been captured.
```

Comment:
```suggestion
The OpenSearch Dashboards release issue will be assigned to the RM who will prepare for the release by reviewing all listed tasks. They should also compare the current release issue to the issue of the previous release version to ensure that all new processes have been captured.
```

also, what is a process in `all new processes`?

Reply:
The release process is evolving with efforts from contributors. Couple of examples - 1. build pipeline will create the release tag automatically instead of manually 2. RM can create release notes based on CHANGELOG in v2.4.0